golang/cosmos/app/app.go

```diff
@@ -838,6 +838,12 @@ func unreleasedUpgradeHandler(app *GaiaApp, targetUpgrade string) func(sdk.Conte
 			return mvm, err
 		}
 
+		m := swingsetkeeper.NewMigrator(app.SwingSetKeeper)
+		err = m.MigrateParams(ctx)
+		if err != nil {
+			return fromVm, err
```

Comment:
Why return `fromVm`?  The prior return does `mvm` even on error.  Maybe consider leaving an explanatory comment?

Reply:
I assumed that when returning an error, the version map should be the "default" value, in this case the "from version map" even though the module manager's migration logic ran its course. I guess it's more a question of what is the best conceptual return value in this case, since if an error is returned, it panics.

Reply:
Use `mvm` since it's less confusing and doesn't matter.